recipes/qwt/all/conandata.yml

```diff
@@ -1,7 +1,9 @@
 sources:
-  "6.1.6":
-    url: "https://sourceforge.net/projects/qwt/files/qwt/6.1.6/qwt-6.1.6.zip"
-    sha256: "4b2452662ae64656aca92f1bef44a281229f77056689100af62c4b9f24462873"
   "6.2.0":
     url: "https://sourceforge.net/projects/qwt/files/qwt/6.2.0/qwt-6.2.0.zip"
     sha256: "3e9632a9be6a883db5c496e42ce74cbbf8da02cc3328faa89e2c43e434a2eb76"
+patches:
+  "6.2.0":
+    - patch_file: "patches/cmake-support.patch"
+      patch_source: "https://github.com/MehdiChinoune/qwt/blob/cmake/CMakeLists.txt"
+    - patch_file: "patches/cmake-support-patch.patch"
```

Comment:
There are thousands lines from an unofficial repo, who will maintain that in CCI?
Was it submitted upstream?

Reply:
It was proposed upstream in https://sourceforge.net/p/qwt/feature-requests/95/#397d/9909
For the reason why this patch is needed, see https://github.com/conan-io/conan-center-index/issues/12830#issuecomment-1237857953

Reply:
In https://sourceforge.net/p/qwt/feature-requests/95/#397d/9909, it's not clear whether these patches will be included in the near future.

Reply:
And in `patch_source`, a link to a commit would be more useful.

Reply:
Indeed, patch_source helps a lot on that case.